scripts/pre_commit_linter.py

```diff
@@ -1218,6 +1216,61 @@ def _check_directive_scope(all_files):
     return summary_messages
 
 
+def _match_line_breaks_in_controller_dependencies(all_files):
+    """This function checks whether the line breaks between the dependencies
+    listed in the controller of a directive or service exactly match those
+    between the arguments of the controller function.
+    """
+    print 'Starting controller dependency check'
```

Comment:
`Starting controller dependency line break check` (otherwise it'll seem that this is checking everything associated with dependencies)

Reply:
Done, thanks!